### ISSUE TITLE
New Imagery for Monongalia County, WV, US

### ISSUE THREAD

**Issue by Jdgat** [osmlab/editor-layer-index#1561]
After a year-ish of requesting from my colleagues that the license terms be updated on our most recent county imagery to allow OSM usage, I have succeeded. Please see below:

## URL of the imagery
https://gis.morgantownwv.gov/image/services/Imagery/Monongalia_County_2022_Aerial_Imagery/ImageServer/WMSServer?request=GetCapabilities&service=WMS

## Two letter country code
US

## Desired layer name
Monongalia County 2022 Aerial Imagery

## License
Custom - "Open Street Map Disclaimer: MMAGIC Associate Members are hereby defined or referred to as 'Associate Members'. MMAGIC is pleased to provide a high-resolution aerial imagery to OpenStreetMap (OSM), in support of its mapping initiatives.  By using the imagery in the OSM editor, users understand and agree that they may only use the imagery to trace and validate edits that must be provided back to OSM. Users may not download the imagery or use the imagery for any other purpose.  Associate Members retain all right, title, and interest in and to the imagery.  The imagery, when made available, is provided “as is”.  Any errors are not the responsibility of MMAGIC; Associate Members disclaim all warranties, express or implied, to the extent permitted by applicable law.  Users may recover from the Associate Members only direct damages up to an amount equal to the fees they have paid to the Associate Members to use the imagery on OSM, if any.  The Associate Members are not liable for any other damages, including consequential, lost-profit, special, indirect, incidental, or punitive damages."

## Is the license compatible with OSM?
Yes

## Attribution (text, URL, required?)
Collected and processed by the Thrasher Group, 2022. 
Owned by the Associate Members of MMAGIC: 
- Monongalia County Commission and related offices
- City of Morgantown
- Morgantown Utility Board

## Privacy policy (URL)
https://www.morgantownwv.gov/privacy/

## Icon
none

## Bounding polygon
https://www.openstreetmap.org/relation/1913126

**Comment by 1ec5** [comment]
[Here’s a privacy policy](https://www.morgantownwv.gov/privacy/) that’s served from the same domain, albeit not the same software.

**Comment by cicku** [comment]
Website down, will handle later.

**Comment by 1ec5** [comment]
Unfortunately, it looks like everything under [the Imagery folder](https://gis.morgantownwv.gov/arcgis/rest/services/Imagery/) is behind a login wall. ([Other services](https://gis.morgantownwv.gov/arcgis/rest/services/) are accessible.) Either this is the wrong URL or the permissions need to be adjusted in order for users to be able to access the iamgery.

**Comment by Jdgat** [comment]
Nice find on that privacy policy - it seems like it was developed for a different sandbox long ago. In general, is that policy OK? I can work to develop a new one for just our GIS.

Yes, seems our server went down. Not a great start! Let me look into it

**Comment by Jdgat** [comment]
Well, looks like the server wasn't actually down after all. Public location of the imagery's overview is here: https://gis.morgantownwv.gov/portal/home/item.html?id=6a31d90f92af4b76b28ac4b1667487aa

Still seems like the actual service is in a non-public folder, however. I'll work with my GIS analyst to update that when he returns to work and post here when we correct it.

**Comment by Jdgat** [comment]
This appears open to public: https://gis.morgantownwv.gov/image/rest/services/Imagery/Monongalia_County_2022_Aerial_Imagery/ImageServer

**Comment by Jdgat** [comment]
OK, so, intricacies of ArcGIS server are at play here... https://enterprise.arcgis.com/en/server/latest/publish-services/windows/communicating-with-a-wms-service-in-a-web-browser.htm

Seems we can get it to work by adding parameters to the URL - maybe getmap is the request we need to make?

**Comment by Jdgat** [comment]
https://gis.morgantownwv.gov/image/services/Imagery/Monongalia_County_2022_Aerial_Imagery/ImageServer/WMSServer?request=GetCapabilities&service=WMS

**Comment by Jdgat** [comment]
I updated the issue with the correct link; sorry for the confusion

**Comment by cicku** [comment]
Can you update your comments at the same place instead of leaving a new one for each update...? I personally don't think it is something urgent so you can sort everything out and update us when it is ready

**Comment by Jdgat** [comment]
@cicku Thank you for the direction, sorry about the excessive email notifications.

**Comment by cicku** [comment]
Please consider using Edit in your comments, OK? People watching the project have received 9 emails/comments.

![image](https://user-images.githubusercontent.com/686438/182848325-e4105539-a312-4471-80cf-40e2cd30b322.png)

**Comment by cicku** [comment]
I'm finding the license text here. Where did you get it?

**Comment by Jdgat** [comment]
@cicku, not sure I understand your question. I wrote it based on the licensing language presented as an example on the OSM wiki. I then presented the language to the consortium of government entities that owns the imagery and they agreed to add it to the imagery service. See language here, in the item details for the WMS: https://gis.morgantownwv.gov/portal/home/item.html?id=f2830b58cd894fa0a0712e2cc9537bba
